### DISCUSSION TITLE
Can I render a common image file using sceneview

### DISCUSSION THREAD

**Discussion by sadrabt** [sceneview/sceneview#315] [Q&A]
I have been playing around with sceneview and I'm able to display glb files. Am I able to use sceneview to place a png/jpeg image? I'm very new to this so I appreciate your help if you could point me to the right place.

**Comment by punkesta** [comment]
Do you find solution?

**Comment by ThomasGorisse** [comment]
https://github.com/SceneView/sceneview-android/blob/main/sceneview%2Fsrc%2Fmain%2Fjava%2Fio%2Fgithub%2Fsceneview%2Fnode%2FImageNode.kt#L21-L34

**Comment by ThomasGorisse** [comment]
Yes! In v4.0.0-rc.1, use `ImageNode`:

```kotlin
SceneView {
    ImageNode(
        materialLoader = materialLoader,
        imageFileLocation = "images/photo.png",
        width = 2.0f,
        height = 1.5f
    )
}
```

`ImageNode` renders images as textured quads in 3D space. Works with PNG, JPG, and other standard formats.
